app/controllers/concerns/hyku_addons/proprietor_controller_override.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+# Adjust controller for our needs
+module HykuAddons
+  module ProprietorControllerOverride
+    extend ActiveSupport::Concern
+
+    def update
+      respond_to do |format|
+        if @account.update(account_params)
+
+          # handles updating of shared search tenant
+          CreateSolrCollectionJob.perform_now(@account)
```

Comment:
I can add a check there since it will make it more obvious but to directly answer you, CreateSolrCollectionJobOverride ensures no action is taken when it is normal  tenant because it check if a solr  collection exists for that account and does nothing if a solr collection exists and it is a normal tenant